### PR TITLE
Alt grid

### PR DESCRIPTION
**Pull request by mhewson**

### CODE REVIEW

File: src/less/definitions/layout/grid.less [hunk 1]
```diff
@@ -58,6 +63,10 @@
     }
 }
 
+.row.row_flex [class^=col-],
+.row.row_flex [class*=col-] {
```
Comment (wjramos) [inline src/less/definitions/layout/grid.less]: we're not really using underscores elsewhere, is this preferred over `.row.flex`, `.row.row-flex`?

Comment (mhewson) [inline src/less/definitions/layout/grid.less]: I'm fine with row-flex for now, but do think we should be moving to BEM over time

Comment (wjramos) [inline src/less/definitions/layout/grid.less]: But this isn't really BEM, is it? 

I thought the standard was:
`.block__element--modifier`

`.grid__row--flex` for example

**Comment by mhewson** [comment]
Yes, I believe you are correct

**Comment by mhewson** [comment]
all good?

**Comment by mhewson** [comment]
approve

[Approved with PullApprove](https://pullapprove.com/rei/rei-cedar/pull-request/74/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=rei/rei-cedar)

**Pull request merged**